Gruntfile.coffee

```diff
@@ -343,7 +343,7 @@ module.exports = (grunt) ->
 					npmInstall: false
 
 		clean:
-			dist: ["dist", "src/base/partials/*sprites*"]
+			dist: ["dist"]
```

Comment:
Is this because we don't want to regenerate the sprites each time? What happens if the sprite map needs to change?

Reply:
This never targeted anything....that path doesn't exist